app/models/user_stats.rb

```diff
@@ -10,7 +10,39 @@ def self.exp_for(user)
     self.stats_for(user).exp
   end
 
+  def activity(date_range = nil)
+    {
+        exercises: {
+            solved_count: organization_exercises
+                              .joins(:assignments)
+                              .where(assignments: { top_submission_status: [:passed, :skipped], submitter: user }.merge_if_present(:submitted_at, date_range))
```

Comment:
This is not necessary to be changed, just a reminder to me to create an issue before merging this.
There are several places where we are doing passed? || skipped? or things like that. We should at least define a method in assignment like `solved_statuses` and use that instead.

Reply:
We also have a lot of code that does the opposite too: `failed_statuses`